tests/hopp/test_hybrid.py

```diff
@@ -2,6 +2,7 @@
 from copy import deepcopy
 
 from pytest import approx, fixture, raises
+import pytest
```

Comment:
Not sure we need this import

Reply:
The `import pytest` is required for using the `subtests` fixture.